From linux-kernel-owner+greg=40kroah.com-S1752514AbWAFRlI@vger.kernel.org Fri Jan  6 09:53:40 2006
Date: Fri, 6 Jan 2006 18:41:01 +0100
From: Adrian Bunk <bunk@stusta.de>
To: <johnpol@2ka.mipt.ru>, <gregkh@suse.de>
Subject: [-mm patch] fix W1_MASTER_DS9490_BRIDGE dependencies
Message-ID: <20060106174101.GT12131@stusta.de>
Content-Disposition: inline

W1_DS9490 was renamed to W1_MASTER_DS9490, but the entry in the 
dependencies of W1_MASTER_DS9490_BRIDGE was forgotten.


Signed-off-by: Adrian Bunk <bunk@stusta.de>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/w1/masters/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- gregkh-2.6.orig/drivers/w1/masters/Kconfig
+++ gregkh-2.6/drivers/w1/masters/Kconfig
@@ -26,7 +26,7 @@ config W1_MASTER_DS9490
 
 config W1_MASTER_DS9490_BRIDGE
 	tristate "DS9490R USB <-> W1 transport layer for 1-wire"
-	depends on W1_DS9490
+	depends on W1_MASTER_DS9490
 	help
 	  Say Y here if you want to communicate with your 1-wire devices
 	  using DS9490R USB bridge.
